common/src/main/java/org/mcaccess/minecraftaccess/features/point_of_interest/LockingHandler.java

```diff
@@ -126,52 +128,64 @@ private void lookAtLockedTarget() {
      */
     private void bowAimingAssist() {
         LocalPlayer player = WorldUtils.getClientPlayer();
+        if (player == null) return;
+
+        // Check if player is using a bow
         if (config.aimAssistEnabled && !aimAssistActive && player.isUsingItem() && player.getUseItem().getItem() instanceof BowItem) {
             List<Entity> hostileEntities = BuiltinEntityPOIGroups.HOSTILE.group.getItems();
             if (!hostileEntities.isEmpty()) {
-                Entity entity = hostileEntities.stream()
-                        .min(Comparator.comparingDouble(e -> WorldUtils.getClientPlayer().distanceTo(e)))
+                Entity closestEntity = hostileEntities.stream()
```

Comment:
Forgot to suggest reverting this variable name previously.
```suggestion
                Entity entity = hostileEntities.stream()
```